code/lib/core-server/src/utils/get-server-channel.ts

```diff
@@ -1,32 +1,52 @@
 import WebSocket, { WebSocketServer } from 'ws';
-import { stringify } from 'telejson';
+import { isJSON, parse, stringify } from 'telejson';
+import type { ChannelHandler } from '@storybook/channels';
+import { Channel } from '@storybook/channels';
 
 type Server = ConstructorParameters<typeof WebSocketServer>[0]['server'];
 
-export class ServerChannel {
-  webSocketServer: WebSocketServer;
+export class ServerChannelTransport {
```

Comment:
Can we add a comment explaining that unlike other channels, this channel can connect to >1 client and a message sent out on the server channel can reach multiple (browser) endpoints?